src/users/integration.test.ts

```diff
@@ -79,13 +79,13 @@ requireAtLeast(
       await expectDave().toHaveProperty('active', true);
 
       // Second activation is a no-op
-      await expect(client.users.db.activate('dynamic-dave')).resolves.toEqual(true);
+      await expect(client.users.db.activate('dynamic-dave')).resolves.toEqual(false);
 
       await client.users.db.deactivate('dynamic-dave');
       await expectDave().toHaveProperty('active', false);
 
       // Second deactivation is a no-op
-      await expect(client.users.db.deactivate('dynamic-dave', { revokeKey: true })).resolves.toEqual(true);
+      await expect(client.users.db.deactivate('dynamic-dave', { revokeKey: true })).resolves.toEqual(false);
 
       await client.users.db.delete('dynamic-dave');
```

Comment:
can you also double check the return code for delete? It should also return `true` if the user was deleted and `false` if the user did not exist at this time

Reply:
addressed in https://github.com/weaviate/typescript-client/pull/285/commits/7d9310ab0621162d4bde50c1c09bbf7c222ef58c